client/src/components/Icon/Facebook.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+import IconFb from "~/assets/icons/social-facebook.svg";
+
+export default (props) => {
+  return <img {...props} src={IconFb}></img>;
```

Comment:
you can probably make it a bit more DRY by having one icon or social-icon component or similar and just render the SVG... for example SVGicon component with svg  / src prop